### DISCUSSION TITLE
Quick Reblog keyboard shortcuts

### DISCUSSION THREAD

**Discussion by contentmint** [AprilSylph/XKit-Rewritten#1332] [Help Desk]
I'm unsure if this functionality still exists, but New XKit used to allow you to use keyboard shortcuts to reblog/draft/queue a post. Is this still possible, and if not, can I put in a request for it?

**Comment by marcustyphoon** [comment]
If you click the tiny floating keyboard icon on the bottom right of the page, you can see a popup with Tumblr's (relatively recently updated!) keyboard shortcuts. Apparently there's none for "draft a reblog of the current post" though:

![image](https://github.com/AprilSylph/XKit-Rewritten/assets/8336245/4259b3e6-93b6-4b5b-ac80-d532b869ff6f)

**Reply by contentmint** [reply]
Yeah, I have seen that helpful guide but I'm afraid I can't find a function that's equivalent to the one the old XKit had. Thanks though!
